hack/generated/pkg/genruntime/errors_test.go

```diff
@@ -112,7 +112,13 @@ func TestOwnerNotFound_RemembersCause(t *testing.T) {
 
 	g.Expect(errors.Cause(err)).To(Equal(cause))
 
-	fmtedErr := fmt.Sprintf("%+v", err)
+	var builder strings.Builder
+	for err != nil {
```

Comment:
I don't think this loop is necessary - the wrapper error should include the context of the errors it wraps. The stacktrace that's needed for the error line assertions can be retrieved using [this technique](https://pkg.go.dev/github.com/pkg/errors?utm_source=godoc#hdr-Retrieving_the_stack_trace_of_an_error_or_wrapper).

Reply:
The `ReferenceNotFound` error wasn't including the nested context, but now it does. I'll query the stack trace separately.

Reply:
Done.